xarray/backends/zarr.py

```diff
@@ -93,7 +93,7 @@ def _determine_zarr_chunks(enc_chunks, var_chunks, ndim, name, safe_chunks):
 
     # if there are no chunks in encoding and the variable data is a numpy
     # array, then we let zarr use its own heuristics to pick the chunks
-    if var_chunks is None and enc_chunks is None:
+    if not var_chunks and not enc_chunks:
```

Comment:
Just for my understanding, could you explain the difference in behavior between `var_chunks is None` and `not var_chunks`?

Reply:
I believe I hit this when var_chunks was an empty list instead of `None`:

```
>>> var_chunks=[]
>>> not var_chunks
True
>>> var_chunks is None
False
>>> 
```

I do not know whey the use case I mention above uses an empty list (while the code was designed to expect None).  I  am not sure if there would be undesired knock-on effects of this change.  Thoughts?

I can still add a whats-new if that is desired.